index.html

```diff
@@ -296,7 +296,111 @@ <h2>The VC HTTP API</h2>
     <section>
       <h3>Authorization</h3>
       <p>
+As an API built on top of HTTP, the VC HTTP API MAY be protected by any security technology
+that is applicable to such HTTP APIs. It is RECOMMENDED that deployments use standard delegation
+protocols to allow for interoperable access to the functions of the API without divulging end-user
+credentials or details unnecessarily to different parties. It is NOT RECOMMENDED that deployments
+use direct authentication technologies such as HTTP Basic Authentication.
       </p>
+      <p>
+This specification defines interoperable bindings to several standard delegation and authorization
+protocols. A conformant implementation deployed with one of these technologies MUST use these
+profiles and MAY further profile the connections as necessary. Each VC HTTP API endpoint defines
+a an authorization <code>type</code>, <code>actions</code>, <code>datatypes</code>, and other 
+security values relevant to the different actions that can be taken on an endpoint.
+These security values are combined and encoded in different ways for the associated security
+protocol, allowing a client to request and a policy to specify fine-grained access control
+for the VC HTTP API endpoints under protection.
+      </p>
+      <section>
+        <h4>OAuth 2.0</h4>
+        <p>
+When calling a VC HTTP API using OAuth 2.0, the request MUST be made using 
+<a href="https://tools.ietf.org/html/draft-ietf-oauth-rar">OAuth 2.0 Rich Authorization Requests</a>
+to describe the rights requested for the given API. The security parameters relevant for
+the rights requested for the endpoints in question are encoded as separate objects within
+the authorization details array.
```

Comment:
I'm not sure I've used the right markup for the new paragraph here.... Also, I don't know whether it would be better to point to DID Registries, VC Registries, or somewhere else.
```suggestion
When making a VC API call using OAuth 2.0, the request SHOULD describe the rights
requested using <a href="https://tools.ietf.org/html/draft-ietf-oauth-rar">OAuth 2.0
Rich Authorization Requests</a>. The security parameters relevant for the rights
requested for the endpoints in question are encoded as separate objects within the
authorization details array. 
        </p>
        <p class="advisory">
Details regarding use of other authorization mechanisms may be specified as future
extensions to the VC API, managed through the _____ Registries.
```